openshift/templates/backup/backup-deploy.yaml

```diff
@@ -163,11 +169,16 @@ parameters:
     description: The name assigned to all of the resources defined in this template.
     required: true
     value: backup
+  - name: ROLE
+    displayName: Role
+    description: Role
+    required: true
+    value: backup
```

Comment:
This may be for another PR: just wondering if we want to sync this configuration with the backup-container one and provide the NetworkPolicy that allows it to access dbs in this namespace by default just by adding the `backup=true` label to the backup targets.

Reply:
Can be dealt with in a separate PR.